docs/source/contents/4_draft_mapping.rst

```diff
@@ -1777,10 +1777,644 @@ Decision
+
+Decision
+^^^^^^^^
+
+Do not migrate.
+
+Here's an `example record - tdh:366 <https://digital.lib.utk.edu/collections/islandora/object/tdh%3A366/datastream/MODS/view>`_.
+
+.. code-block:: xml
+
+    <subject>
+        <topic/>
+    </subject>
+
+Here's an example from `roth - roth:1587 <https://digital.lib.utk.edu/collections/islandora/object/roth%3A1587/datastream/MODS/view>`_.
+
+.. code-block:: xml
+
+    <subject>
+        <name authority="" valueURI="">
+            <namePart/>
+            </name>
+    </subject>
+
+Topical and name subjects with URIs
+-----------------------------------
+
+Use Case
+^^^^^^^^
+
+Remediated collections include subject values with URIs.
+
+Justification
+^^^^^^^^^^^^^
+
+In migration, subjects with name and topical values will be treated in the same way. We have decided that the previous
+distinction between name and topical values as subjects is not essential - only the presence of all the values in the
+metadata is important.
+
+Xpath
+^^^^^
+
+Note that there is inconsistency in where the valueURI attribute is placed.
+
+    :code:`mods:subject[@valueURI]/mods:topic` OR
+    :code:`mods:subject/mods:topic[@valueURI]` OR
+    :code:`mods:subject[@valueURI]/mods:name/mods:namePart` OR
+    :code:`mods:subject/mods:name[@valueURI]/mods:namePart`
+
+Decision
+^^^^^^^^
+
+When a valueURI is present for topical or name subject, it will be the value used in migration. Examples showing each
+of the distinct Xpaths are given below:
+
+`acwiley:280 as an example of mods:subject[@valueURI]/mods:topic <https://digital.lib.utk.edu/collections/islandora/object/acwiley%3A280/datastream/MODS/view>`_
+
+.. code-block:: xml
+
+    <subject authority="lcsh" valueURI="http://id.loc.gov/authorities/subjects/sh85147554">
+        <topic>Women in art</topic>
+    </subject>
+    <subject authority="lcsh" valueURI="http://id.loc.gov/authorities/subjects/sh85147447">
+        <topic>Women artists</topic>
+    </subject>
+    <subject authority="tgm" valueURI="http://id.loc.gov/vocabulary/graphicMaterials/tgm008085">
+        <topic>Portraits</topic>
+    </subject>
+
+.. code-block:: turtle
+
+    @prefix dcterms: <http://purl.org/dc/terms/> .
+
+    <https://example.org/objects/1> dcterms:subject "http://id.loc.gov/authorities/subjects/sh85147554" ;
+        dcterms:subject "http://id.loc.gov/authorities/subjects/sh85147447" ;
+        dcterms:subject "http://id.loc.gov/vocabulary/graphicMaterials/tgm008085" .
+
+`cdf:5384 as an example of mods:subject
+/mods:topic[@valueURI] <https://digital.lib.utk.edu/collections/islandora/object/cdf%3A5384/datastream/MODS/view>`_
+
+.. code-block:: xml
+
+    <subject>
+        <topic valueURI="http://id.loc.gov/authorities/subjects/sh85023396">Child welfare</topic>
+    </subject>
+
+.. code-block:: turtle
+
+    @prefix dcterms: <http://purl.org/dc/terms/> .
+
+    <https://example.org/objects/1> dcterms:subject "http://id.loc.gov/authorities/subjects/sh85023396" .
```

Comment:
Same question regarding double quotes vs angle brackets.

Reply:
Yes, I've apparently polluted half of my analysis with this! Addressing now.

Reply:
Again, :100:. :clap: 

We must check for this throughout this document.